### PR TITLE
[EP-882] Unit tests coverage

### PR DESCRIPTION
**Pull request by AndreiGalkin**
- Jest configuration for ReactNative SDK
- Script `yarn test` for running unit tests and creating coverage reports
- Opens coverage report via default browser by unit testing completion
- Dummy unit test
- adding SDKs yarn.lock into `.gitignore`
- removing tests from sample-app

### CODE REVIEW

File: __tests__/JudoPayTest.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import React from 'react'
+import JudoPay from '../JudoPay';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
+
+test('JudoPay after init has isSandboxed = true', () => {
+  const judoPay = new JudoPay("ta", "atsh")
+  expect(judoPay.isSandboxed).toBe(true);
```
Comment (mpetrenco) [inline __tests__/JudoPayTest.tsx:12]: Won't this fail though?
`isSandboxed` isn't set to `true` by default. Or is this done on purpose to showcase a failed scenario?

Comment (AndreiGalkin) [reply __tests__/JudoPayTest.tsx:12]: As I can see isSandboxed is set true right after var init.

Comment (AndreiGalkin) [reply __tests__/JudoPayTest.tsx:12]: Maybe it's a bug, so we should make it false by default

Comment (mpetrenco) [reply __tests__/JudoPayTest.tsx:12]: image

Comment (mpetrenco) [reply __tests__/JudoPayTest.tsx:12]: We set it to `true` in the sample app

Comment (AndreiGalkin) [reply __tests__/JudoPayTest.tsx:12]: Screenshot 2020-04-16 14 05 52

Comment (mpetrenco) [reply __tests__/JudoPayTest.tsx:12]: 😮 
Ok, my mistake. All good 👍

File: __tests__/JudoPayTest.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// @flow
+import 'react-native'
+import React from 'react'
+import JudoPay from '../JudoPay';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
```
Comment (stpopa) [inline __tests__/JudoPayTest.tsx:8]: extra line?

Comment (AndreiGalkin) [reply __tests__/JudoPayTest.tsx:8]: 👍

**Pull request merged**